web/packages/teleport/src/JoinTokens/JoinTokens.tsx

```diff
@@ -335,8 +394,8 @@ function TokenDelete({
             {` ${token.safeName}`}
           </Text>
           . This will not remove any resources that used this token to join the
-          cluster. This will remove the ability for any new resources to join
-          with this token and any non-renewable resource from renewing.
+          cluster. This will remove the ability for any new resources to join or
+          renew with this token and any non-renewable resource from renewing.
```

Comment:
"to join or renew with this token" - the "or renew with this token" part sounds a bit weird to me

Reply:
I'll remove as "renew" is kind of a join anyway